src/ZohoDatabaseCopier.php

```diff
@@ -70,12 +69,12 @@ public function setLogger(LoggerInterface $logger)
      * @param AbstractZohoDao $dao
      * @param bool            $incrementalSync Whether we synchronize only the modified files or everything.
      */
-    public function copy(AbstractZohoDao $dao, $incrementalSync = true, $twoWaysSync = true)
+    public function copy(AbstractZohoDao $dao, $incrementalSync = true, $twoWaysSync = true, $forceCreateTrigger = false)
     {
         if ($twoWaysSync === true) {
             $this->localChangesTracker->createTrackingTables();
         }
-        $this->synchronizeDbModel($dao, $twoWaysSync);
+        $this->synchronizeDbModel($dao, $twoWaysSync, $forceCreateTrigger);
```

Comment:
The `synchronizeDbModel` is in the ZohoDatabaseCopier class, but come to think about it, it is as much needed by the sync part than by the Zoho part. Maybe we should put it in its own class?